docs/develop/python/temporal-clients.mdx

```diff
@@ -139,7 +139,38 @@ if __name__ == "__main__":
 
 ## Connect to Temporal Cloud {#connect-to-temporal-cloud}
 
-**How to connect to Temporal Cloud using the Python SDK**
+### How to connect to Temporal Cloud using an API key {#connect-to-temporal-cloud-api-key}
+
+To use an [API key](/cloud/api-keys) with the Temporal Python SDK, you will need to gather some information first:
+
+- When you create your _API Key,_ be sure to save the value.
+- You can see the _namespace account id_ combination in your URL or anywhere that you manage namespaces.
+  It follows the format `<namespace_id>.<account_id>`.
+- The _endpoint_ will vary. The most common endpoint used is the gRPC regional endpoint, which you can find by selecting the Connect button on the namespace details page.
+  It follows the format: `<region>.<cloud_provider>.api.temporal.io:7233`.
+- For Namespaces with High Availability features with API key authentication enabled, use the gRPC Namespace endpoint: `<namespace>.<account>.tmprl.cloud:7233`.
+  This allows automated failover without needing to switch endpoints.
+
+Once you have the necessary info, to create an initial connection, use this code:
+
+To create an initial connection:
+
+```python
+client = await Client.connect(
+    <endpoint>,
+    namespace=<namespace_id>.<account_id>,
+    api_key=<APIKey>,
+    tls=True,
+)
+```
+
+To update an API key:
```

Comment:
This is kind of a nonsense example to provide (and it also, very confusingly, says `my_client` when the previous sample used `client`). I would just say:

```suggestion
To update an API key, you can replace the value of `api_key` in the `client` object as needed.
```